Herebyfile.mjs

```diff
@@ -0,0 +1,100 @@
+import {copyFile, mkdir, readFile, writeFile} from 'fs/promises';
+
+import {execa} from 'execa';
+import {task} from 'hereby';
+import semver from 'semver';
+
+import {docgen, spliceIntoSection} from '@puppeteer/docgen';
+
+export const docsNgSchematicsTask = task({
+  name: 'docs:ng-schematics',
+  run: async () => {
+    const readme = await readFile('packages/ng-schematics/README.md', 'utf-8');
+    const index = await readFile('docs/integrations/ng-schematics.md', 'utf-8');
+    await writeFile(
+      'docs/integrations/ng-schematics.md',
+      index.replace('# API Reference\n', readme)
+    );
+  },
+});
+
+export const docsChromiumSupportTask = task({
+  name: 'docs:chromium-support',
+  run: async () => {
+    const content = await readFile('docs/chromium-support.md', {
+      encoding: 'utf8',
+    });
+    const {versionsPerRelease} = await import('./versions.js');
+    const versionsArchived = JSON.parse(
+      await readFile('website/versionsArchived.json', 'utf8')
+    );
+    const buffer = [];
+    for (const [chromiumVersion, puppeteerVersion] of versionsPerRelease) {
```

Comment:
let's include https://github.com/puppeteer/puppeteer/pull/11108 into this change or as a follow-up? the contributor's CL would go stale with this one.

I have also asked @Lightning00Blade to review as he said he spotted something.